src/app/sharedViews/layout.ejs

```diff
@@ -116,9 +109,10 @@
     </div>
 </footer>
 
-<script src="<%= locals.urls.assets %>/javascript/vendors/jquery.min.js"></script>
-<script src="<%= locals.urls.assets %>/javascript/vendors/select2.min.js"></script>
-<script src="<%= locals.urls.assets %>/javascript/app.min.js"></script>
+<script src="<%= locals.urls.assets %>/javascript/vendors/jquery.min.js?version=<%= locals.assets.version %>"></script>
+<script src="<%= locals.urls.assets %>/javascript/vendors/select2.min.js?version=<%= locals.assets.version %>"></script>
+<script src="<%= locals.urls.assets %>/javascript/app.min.js?version=<%= locals.assets.version %>"></script>
```

Comment:
we also need [<%= locals.urls.assets %>/javascript/vendors/vendors.min.js](https://github.com/DFE-Digital/login.dfe.services/blob/develop/src/app/layouts/layout.ejs#L153)

Reply:
in that case we get the cookie banner but we said we were not adding it to support?

Reply:
GA utils are also in [vendors.min.js](https://github.com/DFE-Digital/dfe.ui.toolkit/blob/master/src/javascript/vendors.js)